figma/emoji-mart
# Update font family for native emoji previews

As explained in https://github.com/figma/figma/pull/149071, our default font-family was rendering some emojis (like 1️⃣) incorrectly. This adds a font-family style attribute to the native emoji previews in the picker component.

Before
<img width="415" alt="Screen Shot 2023-05-22 at 10 49 02 AM" src="https://github.com/figma/emoji-mart/assets/12766907/c6d74a08-f966-466a-af41-2503fa050e50">

After
<img width="364" alt="Screen Shot 2023-05-22 at 1 18 22 PM" src="https://github.com/figma/emoji-mart/assets/12766907/75e3fc89-90b4-4b2a-a19e-37686cd24a98">

## Test plan
- Modify `web/package.json` to use the latest commit on this branch for the emoji-mart package
- Run `npm install && npm run build`
- Make comment in a local Figma file and open the emoji picker
- All comments should render properly
